theano/tests/test_flake8.py

```diff
@@ -331,15 +325,19 @@
     "gof/tests/test_link.py",
     "gof/tests/test_fg.py",
     "gof/tests/test_sched.py",
-    "gof/tests/test_op.py",
     "gof/tests/test_graph_opt_caching.py",
     "gof/tests/test_graph.py",
     "gof/tests/test_cc.py",
     "gof/tests/test_compute_test_value.py",
     "gof/sandbox/equilibrium.py",
-    "generated_version.py",
     "sandbox/cuda/opt_util.py",
     "gof/tests/test_utils.py",
+    "printing.py",
+    "raise_op.py",
+    "tests/test_flake8.py",
+    "misc/pkl_utils.py",
+    "sandbox/cuda/tests/test_cula.py",
+    "tensor/tests/_test_mpi_roundtrip.py",
```

Comment:
Yes, they're needed in the whitelist when forcing the new `pep8`. They aren't in there right now because Travis was actually using an old `pep8` version.

I'd be happy to look at fixing them, but I can do that in a later PR? I'd hate for endless merge conflicts to occur in the meantime.